api/sourcecontrol/v1/sourcecontrol.proto

```diff
@@ -21,6 +29,20 @@ service SourceControlAPI {
   }
 }
 
+message GetRepositoryOptionsRequest {
+}
+
+// An entity encompasses members and organizations who are both owners of or contributors to source code.
+message Entity {
+  string name = 1;
+  string photo_url = 2;
+}
+
+message GetRepositoryOptionsResponse {
+  repeated Entity available_owners = 1;
+  repeated clutch.config.module.sourcecontrol.v1.Visibility visibility_options = 2;
```

Comment:
i don't think we've run into this before, but config objects should not leak over to the RPC side. i think the converse doesn't necessarily need to be true so you could move visibility_options over to the RPC side and then reference it on the config side.

one reason for this is that at some point we want to stop bundling service config protos over to the frontend where they are unused.